tests/labhub_test.py

```diff
@@ -142,8 +147,9 @@ def test_create_issue_cmd(self):
         # TODO
         # Ignoring assertion to prevent build failure for time being
         # Creating issue in private chat
-        # testbot_private.assertCommand('!new issue repository this is the title\nbo\ndy',
-        #                       'You\'re not allowed')
+        # testbot_private.assertCommand('!new issue repository this '
```

Comment:
Using `Ignore bear` here, as this line will be modified again soon when you fix the bug, and will then be reformatted.